bndtools.m2e/src/bndtools/m2e/MavenBndrunContainer.java

```diff
@@ -41,10 +44,19 @@
 	private static final ProjectRegistryManager			projectManager			= MavenPluginActivator.getDefault()
 		.getMavenProjectManagerImpl();
 
-	public BndrunContainer getBndrunContainer(IMavenProjectFacade projectFacade, MojoExecution mojoExecution,
+	private static final MavenRunListenerHelper			helper					= new MavenRunListenerHelper() {};
```

Comment:
If this class already implements `MavenRunListenerHelper`, why does it need another `helper` instance? Is it because you need to call from static methods? Perhaps this class should no longer implement `MavenRunListenerHelper`?

Reply:
It's because the methods are static. Do note that I'm cleaning all this up in a future PR.